ikasaneip/sample/spring-boot/file/src/test/java/com/ikasan/sample/spring/boot/builderpattern/ApplicationTest.java

```diff
@@ -115,27 +122,35 @@
     @DirtiesContext
     public void sourceFileFlow_flow() throws Exception
     {
+        flowTestRule.withFlow((Flow) moduleUnderTest.getFlow("sourceFileFlow"));
+
         // create file to be consumed
         String content = "Hello World !!";
         Files.write(Paths.get(FILE_CONSUMER_FILE_NAME), content.getBytes());
 
-        Flow flow = (Flow) moduleUnderTest.getFlow("sourceFileFlow");
 
-        ConfiguredResource configuredResource = ((ConfiguredResource)flow.getFlowElement("File Consumer").getFlowComponent());
+        ConfiguredResource configuredResource = (ConfiguredResource)flowTestRule.getComponent("File Consumer");
         Object configuration = configurationManagement.createConfiguration(configuredResource);
```

Comment:
As previous, why not pull from component rather than create a new configuration and saving it?